strangerstudios/pmpro-member-directory
# Adds support for user taxonomies on directory and profile

### All Submissions:






### Changes proposed in this Pull Request:

Adds support for custom taxonomies so that the tax ID does not show on the profile/directory pages but rather the taxonomy name. 

### How to test the changes in this Pull Request:

1. Add this recipe to set up multiple custom taxonomies https://github.com/strangerstudios/pmpro-snippets-library/blob/dev/user-fields/custom-user-taxonomy-field.php
2. Directory Shortcode: [pmpro_member_directory fields="Team,team;Title,titles;Rev,revenue;"]
3. Profile Shortcode: [pmpro_member_profile fields="Team,team;Title,titles;Rev,revenue;"]
4. Update a user profile with the new taxonomies
5. The taxonomy labels/names should show and not their stored IDs on both the directory and profile pages.

### Other information:




### Changelog entry

Custom user taxonomies now display labels instead of stored IDs
